### ISSUE TITLE
Bug in vs 2019 16.1.0 preview 2 when commenting Razor page function

### ISSUE THREAD

**Issue by darren-zdc** [aspnet/AspNetCore#9723]
### Describe the bug
If I comment selection of code in razor file function, vs will crash and reopen. No idea what is causing it,

Environment:
1. dotnet core sdk 3.0 preview 4
2. vs 2019 16.1.0 preview 2.0

### To Reproduce
1. Open a new Blazor server-side project
2. Use `Ctrl K + Ctr C`  or 
![image](https://user-images.githubusercontent.com/36833304/56685669-7da0c700-66a0-11e9-9bb1-3d357409c0bc.png)
to comment selection of code inside `Counter.razor` file
![image](https://user-images.githubusercontent.com/36833304/56685381-d459d100-669f-11e9-9df8-8d34cf47b308.png)

### Expected behavior
1. vs will crash and reopen

**Comment by NTaylorMullen** [comment]
Can reproduce this. It be bad. Thanks for the report @darren-zdc!

**Comment by BenHayat** [comment]
Seems like related to the existing extension.

Here the error log file.

[ActivityLog.zip](https://github.com/aspnet/AspNetCore/files/3113981/ActivityLog.zip)

**Comment by ajaybhargavb** [comment]
Investigated this. Looks like the crash happens in VS.Platform.VSEditor
```
System.InvalidOperationException
  HResult=0x80131509
  Message=Unable to get TextViewLine containing insertion point.
  Source=Microsoft.VisualStudio.Platform.VSEditor
  StackTrace:
   at Microsoft.VisualStudio.Text.MultiSelection.Implementation.SelectionUIProperties.get_ContainingTextViewLine()
   at Microsoft.VisualStudio.Text.MultiSelection.Implementation.SelectionUIProperties.get_IsWithinViewport()
   at Microsoft.VisualStudio.Text.MultiSelection.Implementation.UI.AccessibleCaretElementAutomationAdapter.IsOffscreenCore()
   at System.Windows.Automation.Peers.AutomationPeer.UpdateSubtree()
   at System.Windows.Automation.Peers.AutomationPeer.UpdateSubtree()
   at System.Windows.Automation.Peers.AutomationPeer.UpdateSubtree()
   at System.Windows.Automation.Peers.AutomationPeer.UpdateSubtree()
   at System.Windows.Automation.Peers.AutomationPeer.UpdateSubtree()
   at System.Windows.Automation.Peers.AutomationPeer.UpdateSubtree()
   at System.Windows.Automation.Peers.AutomationPeer.UpdateSubtree()
   at System.Windows.Automation.Peers.AutomationPeer.UpdateSubtree()
   at System.Windows.Automation.Peers.AutomationPeer.UpdateSubtree()
   at System.Windows.Automation.Peers.AutomationPeer.UpdateSubtree()
   at System.Windows.Automation.Peers.AutomationPeer.UpdateSubtree()
   at System.Windows.ContextLayoutManager.fireAutomationEvents()
   at System.Windows.ContextLayoutManager.UpdateLayout()
   at System.Windows.ContextLayoutManager.UpdateLayoutCallback(Object arg)
   at System.Windows.Media.MediaContext.InvokeOnRenderCallback.DoWork()
   at System.Windows.Media.MediaContext.FireInvokeOnRenderCallbacks()
   at System.Windows.Media.MediaContext.RenderMessageHandlerCore(Object resizedCompositionTarget)
   at System.Windows.Media.MediaContext.RenderMessageHandler(Object resizedCompositionTarget)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
```

**Comment by NTaylorMullen** [comment]
Spoke to @Amadeusw and he's forwarding this onto the proper folks. I'll update this issue accordingly based on that convo.

**Comment by Andrzej-W** [comment]
I can confirm I see the same behaviour. I can use `Ctrl K + Ctrl C` to comment razor (html) code, but VS crashes when I try to comment something in `@functions` block.

**Comment by NTaylorMullen** [comment]
This was fixed by Roslyn in VS 16.1-Preview4. https://github.com/dotnet/roslyn/issues/35285
